schema/set-protection.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SetProtectionParams",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["isProtected", "eventOrigin"],
+    "properties": {
+        "isProtected": {
+            "type": "boolean"
+        },
+        "eventOrigin": {
+            "$ref": "event-origin.json"
+        }
+    }
+}
```

Comment:
this existed before, but wasn't in the schema since we just sent a 'boolean', but now we're sending an object